File: Python/Logic-1/near_ten.py
"""
Dada um número inteiro "num", retorne true caso ele esteja 2 unidades próximas
de ser um múltiplo de 10.
"""


def near_ten(num):
    return num % 10 <= 2 or num % 10 >= 8


print(near_ten(10))

File: Java/Logic-1/nearTen/NearTen.java
package nearTen;

public class NearTen {
    //Retorne true se o número não negativo dado estiver no intervalo de 2 unidades maior/menor que um múltiplo de 10.

    public boolean nearTen(int num) {
        return (num%10<=2 || num%10>=8);
    }
}
